docs/introduction/dev-environment-quick-start-guide.md

```diff
@@ -289,4 +289,23 @@ docker-compose up
 ```
 > **Note:**
 > To know more about docker-compose commands do read [this guide](../how-to-guides/docker-developer-guide.md) 
-  
+
+### make dev error: [build_lang] Error 13 - can't write into /mnt/podata/data/Lang.openfoodfacts.localhost.sto
+
+When running `make dev`:
+
+```console
+<h1>Software error:</h1>
+<pre>can't write into /mnt/podata/data/Lang.openfoodfacts.localhost.sto: Permission denied at /opt/product-opener/lib/ProductOpener/Store.pm line 234.
+</pre>
+<p>
+For help, please send mail to this site's webmaster, giving this error message
+and the time and date of the error.
+
+</p>
+make: *** [Makefile:126: build_lang] Error 13
+```
+
+**Solution:**
+
+Use the powershell/cmd to run the make dev commands in windows.
```

Comment:
I'm just adding a Makefile warning so that people use git bash console ! (It's too hard to support powershell/cmd using Makefile, and git bash comes with git).

```suggestion
Use the git bash console to run the make dev commands in windows.
```

Reply:
That's the problem @alexgarel when I use gitbash for "make dev" it's throw that error so, I have to switch to cmd to run "make dev"

Reply:
Hum we have to investigate then… I just merged the PR which forces bash.
Can you try again (but using git bash) ?

Reply:
Nope! same result.